src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -706,10 +703,10 @@ Json::Value checkFee (
 
     if (fee > limit && fee != loadFee &&
         request.isMember("x-queue-okay") &&
-            request["x-queue-okay"].isBool() &&
+            request["x-queue-okay"].isConvertibleTo(Json::booleanValue) &&
```

Comment:
Just noting that `isConvertibleTo()` is pretty permissive and even allows reals: https://github.com/ripple/rippled/blob/develop/src/ripple/json/impl/json_value.cpp#L712.  Are you sure you want to be that permissive?